pkg/api/openshiftcluster.go

```diff
@@ -312,6 +312,57 @@ const (
 	VMSizeStandardL64sV2 VMSize = "Standard_L64s_v2"
 )
 
+type AzureVM struct {
+	Type      VMSize
+	CoreCount int
+	Family    string
+}
+
+var vmTypesMap map[VMSize]AzureVM = map[VMSize]AzureVM{
```

Comment:
In this package we define internal representation of our objects. This code is validation-specific. It should not be here.